src/data/resolvers/motions.ts

```diff
@@ -158,6 +158,28 @@ export const motionsResolvers = ({
       const timeToSubmitMS = motion.events[1].toNumber() * 1000;
       const timeToSubmitInPast = timeToSubmitMS < blocktime;
 
+      const latestMotionStakedEvent = sortedEvents.find(
+        (event) => event.name === ColonyAndExtensionsEvents.MotionStaked,
+      );
+
+      if (latestMotionStakedEvent) {
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        const requiredStake = getMotionRequiredStake(
+          motion.skillRep,
+          totalStakeFraction,
+          18,
+        );
+
+        if (motion.stakes[MotionVote.Yay].eq(requiredStake)) {
```

Comment:
It might also be bigger by 1 than the required stake _(due to a bug in the contracts)_ so the easiest way around that is to check if greater than or equal

```suggestion
        if (motion.stakes[MotionVote.Yay].gte(requiredStake)) {
```

Reply:
As I've stated above, here's the test for this:

Motion is fully staked on the yay side:
![Screenshot from 2021-04-26 10-29-55](https://user-images.githubusercontent.com/1193222/116045218-8145fa80-a67a-11eb-962b-62b206670c1f.png)

But the actual BigNumber values you are comparing here are off by one:
![Screenshot from 2021-04-26 10-31-17](https://user-images.githubusercontent.com/1193222/116045311-9884e800-a67a-11eb-864b-48df25590c09.png)